static/js/zcommand.js

```diff
@@ -60,6 +60,7 @@ function update_setting(command) {
 exports.process = function (message_content) {
 
     var content = message_content.trim();
+    var tokens = content.split(' ');
```

Comment:
A better name for `tokens` can be suggested.

Reply:
`tokens` seems fine here.